test/testSend.js

```diff
@@ -25,7 +25,7 @@ const { setupMocha: setupPolly } = require('@pollyjs/core');
 const env = require('../src/env.js');
 
 function getQuery(replacer) {
-  return `--- Authorization: none
+  return `/*\ngood commenting is encouraged\n*/\n--- Authorization: none
```

Comment:
You can use line breaks instead of `/n` in backtick-enclosed template literals. That makes everything more readable.